lib/ecto_mnesia/record/context/match_spec.ex

```diff
@@ -29,9 +29,9 @@ defmodule EctoMnesia.Record.Context.MatchSpec do
 
   # Build match_spec head part (data placeholders)
   defp match_head(%Context{table: %Context.Table{name: table_name}} = context) do
-    context
-    |> Context.get_fields_placeholders()
-    |> Enum.into([table_name])
+    [ table_name
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.